src/gui/screen_menu_lan_settings.cpp

```diff
@@ -23,7 +23,14 @@ ScreenMenuConnectionBase::ScreenMenuConnectionBase(uint32_t dev_id, const char *
 
 void ScreenMenuConnectionBase::refresh_addresses() {
     char str[ADDR_LEN];
-
+    char fwver[4];
+    const uint8_t fw_version = espif_firmware_version();
```

Comment:
You can put this inside the if statement: `if(const auto fw_version = espif_firmware_version())`

Reply:
Done.